static/app/views/dashboards/widgets/timeSeriesWidget/releaseSeries.tsx

```diff
@@ -14,9 +14,11 @@ export function ReleaseSeries(
   releases: Release[],
   onClick: (release: Release) => void,
   utc: boolean
-): Series {
+): CustomSeriesOption {
   return {
-    seriesName: t('Releases'),
+    type: 'custom',
+    name: t('Releases'),
+    renderItem: () => null,
```

Comment:
We were using `LineSeries`, but it's not really a line series since lines are drawn using mark lines. I've changed it to use a custom series is that people don't conflate the mark lines + line series.